docs/hive.md

```diff
@@ -244,7 +254,7 @@ The result is:
 | j                                  | IDENTITY       | NULL
 
 You can create Iceberg partitions using the following Iceberg partition specification syntax
-(supported only in Hive 4.0.0-alpha-1):
+(supported only from Hive 4.0.0-alpha-1):
```

Comment:
How does this create partitions? Is this trying to say you can alter partition fields?

Reply:
I wouldn't say that. It is about creating a table, not altering one.